resources/ingest_demo_data.sh

```
@@ -0,0 +1,902 @@
+        
+        # Execute curl
+        local response
+        response=$(curl "${curl_args[@]}" 2>&1)
+        local curl_exit_code=$?
+        
+        # Check curl exit code
+        if [[ $curl_exit_code -eq 0 ]]; then
+            local status_code
+            if [[ -n "$response" ]]; then
+                status_code=$(echo "$response" | tail -n1)
+                local response_body=$(echo "$response" | sed '$d')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)